scripts/kind-e2e/e2e.sh

```diff
@@ -212,6 +212,7 @@ function kind_import_images() {
         echo "Loading submariner images in to cluster${i}..."
```

Comment:
I'm getting an error when trying to build/deploy the latest submariner repo with this patch. 
The following seems to be missing in Line200
docker tag quay.io/submariner/submariner-ipam:dev submariner-ipam:local